aspnetcore/blazor/hosting-models.md

```diff
@@ -111,20 +119,31 @@ For more information on Microsoft native client frameworks, see the following re
+
+Technically, a Blazor Progressive Web Application (PWA) is a form of Blazor WebAssembly and not a separate hosting model. However, Blazor PWA offers a unique combination of features, so PWA appears in the following table in its own column.
 
-| Feature | Blazor Server | Blazor WebAssembly (WASM) | Blazor Hybrid |
-| --- | :---: | :---: | :---: |
-| [Complete .NET API compatibility](#complete-net-api-compatibility) | ✔️ | ❌ | ✔️ |
-| [Direct access to server and network resources](#direct-access-to-server-and-network-resources) | ✔️ | ❌&dagger; | ❌&dagger; |
-| [Small payload size with fast initial load time](#small-payload-size-with-fast-initial-load-time) | ✔️ | ❌ | ❌ |
-| [App code secure and private on the server](#app-code-secure-and-private-on-the-server) | ✔️ | ❌&dagger; | ❌&dagger; |
-| [Run apps offline once downloaded](#run-apps-offline-once-downloaded) | ❌ | ✔️ | ✔️ |
-| [Static site hosting](#static-site-hosting) | ❌ | ✔️ | ❌ |
-| [Offloads processing to clients](#offloads-processing-to-clients) | ❌ | ✔️ | ✔️ |
-| [Full access to native client capabilities](#full-access-to-native-client-capabilities) | ❌ | ❌ | ✔️ |
+Blazor Hybrid apps include .NET MAUI, WPF, and Windows Forms framework apps.
 
-&dagger;Blazor WebAssembly and Blazor Hybrid apps can use server-based APIs to access server/network resources and access private and secure app code.
+| Feature | Blazor Server | Blazor WebAssembly (WASM) | Blazor PWA | Blazor Hybrid |
+| --- | :---: | :---: | :---: | :---: |
+| [Complete .NET API compatibility](#complete-net-api-compatibility) | ✔️ | ❌ | ❌ | ✔️ |
+| [Direct access to server and network resources](#direct-access-to-server-and-network-resources) | ✔️ | ❌&dagger; | ❌&dagger; | ❌&dagger; |
+| [Small payload size with fast initial load time](#small-payload-size-with-fast-initial-load-time) | ✔️ | ❌ | ❌&Dagger; | ❌ |
+| [Relatively larger payload size with faster execution speed](#relatively-larger-payload-size-with-faster-execution-speed) | ❌ | ✔️ | ✔️&Dagger; | ✔️ |
+| [App code secure and private on the server](#app-code-secure-and-private-on-the-server) | ✔️ | ❌&dagger; | ❌&dagger; | ❌&dagger; |
+| [Run apps offline once downloaded](#run-apps-offline-once-downloaded) | ❌ | ✔️ | ✔️ | ✔️ |
+| [Static site hosting](#static-site-hosting) | ❌ | ✔️ | ✔️ | ❌ |
+| [Offloads processing to clients](#offloads-processing-to-clients) | ❌ | ✔️ | ✔️ | ✔️ |
+| [Full access to native client capabilities](#full-access-to-native-client-capabilities) | ❌ | ❌ | ❌ | ✔️ |
+| [Executes in its own app window](#executes-in-its-own-app-window) | ❌ | ❌ | ✔️ | ✔️ |
+| [Launch from the client](#launch-from-the-client) | ❌ | ❌ | ✔️ | ✔️ |
+| [Receive push notifications](#receive-push-notifications) | ❌ | ❌ | ✔️ | ✔️ |
+| [Automatic background updates](#automatic-background-updates) | ❌ | ❌ | ✔️ | ✔️ |
+| [Provided to clients via an installer](#provided-to-clients-via-an-installer) | ❌ | ❌ | ❌ | ✔️ |
```

Comment:
Let's flip this one around. The web deployment model is generally considered a strength over native clients.

```suggestion
| [Web-based deployment](#web-based-deployment) | ✔️ | ✔️ | ❌ |
```